mockup/patterns/structure/js/views/app.js

```diff
@@ -125,38 +125,62 @@ define([
         self.loading.show();
         self.updateButtons();
 
+        // the remaining calls are related to window.pushstate.
+        // abort if feature unavailable.
+        if (!(window.history && window.history.pushState)) {
+          return
+        }
+
+        // undo the flag set by popState to prevent the push state
+        // from being triggered here, and early abort out of the
+        // function to not execute the folowing pushState logic.
+        if (self.doNotPushState) {
+          self.doNotPushState = false;
+          return
+        }
+
+        var path = self.getCurrentPath();
+        if (path === '/'){
+          path = '';
+        }
         /* maintain history here */
-        if(self.options.urlStructure && window.history && window.history.pushState){
-          if (!self.doNotPushState){
-            var path = self.getCurrentPath();
-            if(path === '/'){
-              path = '';
-            }
-            var url = self.options.urlStructure.base + path + self.options.urlStructure.appended;
-            window.history.pushState(null, null, url);
-            $('body').trigger('structure-url-changed', path);
-          }else{
-            self.doNotPushState = false;
-          }
+        if (self.options.urlStructure) {
+          var url = self.options.urlStructure.base + path + self.options.urlStructure.appended;
+          window.history.pushState(null, null, url);
```

Comment:
You're not checking for availability of window.history.pushState here, like it was done before. Therefore loosing compatibility with IE < 10. (IIRC, we *wanted* to support IE>=8)